SimpleBook.php

```diff
@@ -1,7 +1,7 @@
 <?php
 
 /**
- * Collection Extension for MediaWiki
+ * SimpleBook Extension for MediaWiki
  *
  * Copyright (C) PediaPress GmbH
```

Comment:
Sorry to drop in late. Do we need to update this?

Reply:
I was wondering and figured I'd ask @kfogel that some time / do that on a future PR if so.

Hey @kfogel do we need to update that?

Reply:
Yes, we should update that.  Now, we should also be clearly documenting (which I think we are?) in the top-level README that this is descended from Collection.  We want to be sure to keep provenance and proper attribution and all that.

Also when we modify a file, we should update the Copyright line to name OTS as well, e.g.,:

`Copyright (C) PediaPress GmbH; Open Tech Strategies, LLC`

Reply:
(For any new files that we've added, the copyright line should just name OTS, of course.)

Reply:
Linking in our concurrent [Zulip conversation about this](https://chat.opentechstrategies.com/#narrow/stream/2-general/topic/All.20Things.20open/near/111680).